externals/llvm-external-projects/torch-mlir-dialects/lib/Dialect/Tcp/IR/TcpOps.cpp

```diff
@@ -19,6 +19,33 @@
 #include "torch-mlir-dialects/Dialect/Tcp/IR/TcpOps.cpp.inc"
 
 namespace mlir::tcp {
+
+LogicalResult ClampOp::verify() {
+  auto inputType = getIn().getType().dyn_cast<RankedTensorType>();
```

Comment:
Isn't this already verified by the type constraint you use ODS side?

Reply:
Yes, good point. Removed this check.